src/auction.vy

```diff
@@ -50,62 +135,44 @@ def _transfer_ownership(new_owner: address):
     log OwnershipTransferred(old_owner, new_owner)
 
 
+
 # ///////////////////////////////////////////////////// #
 #                  Auction House Logic                  #
 # ///////////////////////////////////////////////////// #
 
+
+
 event AuctionStarted:
     lot: indexed(uint256)
     patron: indexed(address)
+    endDate: uint256
 
 
 event AuctionEnded:
     lot: indexed(uint256)
     winner: indexed(address)
+    proceeds: uint256
 
 
 event BidSubmitted:
     lot: indexed(uint256)
     bidder: indexed(address)
     bid: uint256
+    newEndDate: uint256
 
 
-bid_token: public(immutable(ERC20))
-nft: public(immutable(ERC721))
-
 topBid: public(HashMap[uint256, Bid])
 patron: public(HashMap[uint256, address])
 auction_ends: public(HashMap[uint256, uint256])
 
-auction_duration: public(uint256)
 
-fee: public(uint256)
 
 profit: public(uint256)
```

Comment:
Is this terminology acceptable to the protocol?

Reply:
ack: renamed to protocol_fee